docs/src/reference/export.md

```diff
@@ -24,10 +24,16 @@ evaluate_at_grid_nodes
 ```
 
 ## VTK Export
-
 ```@docs
-vtk_grid(filename::AbstractString, grid::Grid{dim,C,T}; compress::Bool) where {dim,C,T} 
-vtk_point_data
-vtk_cellset
-vtk_cell_data_colors
+VTKFile
+VTKFileCollection
```

Comment:
Why not `ParaviewCollection` collection?

Reply:
@fredrikekre and I concluded that `VTKFileCollection` makes the intent/purpose more clear. 
Technically, it is a "paraview datafile", the "paraview collection" is a term that I think is only introduced by `WriteVTK.jl`?

Reply:
I think I am missing something. `.pvd` is one file format to group vtk files in a specific way (which due to paraview the most prominent one) and nothing specific to the WriteVTK.jl package. Another way to group vtk files is for example the `.pvtk` format or the `.xdmf` format. Or is this one not about exporting to paraview anymore?

Reply:
No, pvd (Paraview datafile) is fine, but we just thought that the name "Paraview Collection" is not as descriptive or generally used (outside WriteVTK). But I could be wrong. Initially, I had PVDFile as another alternative.

Reply:
Just that I understand correctly: We then control which type of "grouping" we write by another input argument for the constructor call to `VTKFileCollection`? E.g. `VTKFileCollection("my_output"; type=ParaViewOutput)`?

Reply:
Hm, I haven't thought about that! That sounds like a nice idea!

And for sure that would be fairly easy to extend to if more vtk collection types are desired. 
And probably a good idea to gather all of them (vtk-style) the same type, if possible.
But it only makes sense for time series though, so not for grouping multiple files for the same time step.

Reply:
Thinking about FerriteDistributed.jl - how would we handle a ParaviewCollection with pvtk files in the new abstraction layer?

Reply:
Nice! Yes, that is a good way to check how well it generalizes. 
I think that we should relax the type demands on the `vtk` field in `VTKFile` to `WriteVTK.VTKFile` instead of `WriteVTK.DatasetFile`, to allow `WriteVTK.PVTKFile` as well. Then the `VTKFile` constructor would just need to be overloaded for the `AbstractNODGrid` or (dofhandler), and if needed write* functions too. 

This should make the collection work automatically. Not sure how the distributed files work though... Do they all get the same names and need to be combined at the end somehow?

Do you have an example with the old `paraview_collection` already?

Reply:
Can pvd files not refer to pvtk files?

Reply:
Good point @fredrikekre!
Modifying the pvtk example from WriteVTK seems to work, but perhaps some redundant pointers to different files. Opening `pvd1.pvd` below in Paraview also works fine, but even `pvd2.pvd` seems to contain all the data...
In any case, by using the extra options that would be available when we overload this ourselves in `FerriteDistributed`, I'm quite confident that we would be in an even better situation wrt. making sure it works together for those cases (one option could be to only save the collection on the main processor, and that collection should only refer to the `pvtu` files, which again points to the `vtu` files for each processor at the current time step). 
@termi-official: Do you think this looks to be promising from your point of view?
```julia
all_data = [
    # Process 1
    (
        points = rand(3, 5),  # 5 points on process 1
        cells = [             # 2 cells  on process 1
            MeshCell(VTKCellTypes.VTK_TRIANGLE, [1, 4, 2]),
            MeshCell(VTKCellTypes.VTK_QUAD, [2, 4, 3, 5]),
        ],
    ),

    # Process 2
    (
        points = rand(3, 4),  # 4 points on process 2
        cells = [             # 1 cell   on process 2
            MeshCell(VTKCellTypes.VTK_QUAD, [1, 2, 3, 4]),
        ]
    ),
]

saved_files = Vector{Vector{String}}(undef, 2)  # files saved by each "process"
pvds = [paraview_collection("pvd$i") for i in 1:2]

for (i, t) in enumerate((0.0, 0.2))
    for part = 1:2
        data = all_data[part]
        saved_files[part] = pvtk_grid(
                "simulation$i", data.points, data.cells;
                part = part, nparts = 2,
            ) do pvtk
            pvtk["Pressure"] = sum(data.points; dims = 1)
            pvds[part][t] = pvtk
        end
    end
end
vtk_save.(pvds)
```
gives 
```julia
2-element Vector{Vector{String}}:
 ["pvd1.pvd", "simulation1.pvtu", "simulation1\\simulation1_1.vtu", "simulation2.pvtu", "simulation2\\simulation2_1.vtu"]
 ["pvd2.pvd", "simulation1\\simulation1_2.vtu", "simulation2\\simulation2_2.vtu"]
 ```